custom_components/ramses_cc/translations/en.json

```diff
@@ -0,0 +1,183 @@
+                    "scan_interval": "How often to poll for changes in state. Values below 60 or above 180 are not recommended.",
+                    "ramses_rf": "Not required for typical use."
+                }
+            },
+            "schema": {
+                "title": "System schema and known devices",
+                "description": "Refer to the configuration section in the [wiki](https://github.com/zxdavb/ramses_cc/wiki/) for further details and examples of system schema configuration.",
+                "data": {
+                    "schema": "System schema(s)",
+                    "known_list": "Known device IDs",
+                    "enforce_known_list": "Accept packets from known devices IDs only"
+                },
+                "data_description": {
+                    "schema": "A mapping of system device IDs to their respective schemas. This should be kept minimal and only contain devices that are not automatically discovered. For example:\n```\n01:111111: # Temperature control system (e.g. evohome)\n  system:\n    appliance_control: 10:123446\n  zones:\n    \"07\": '{sensor: 01:111111}'\n\norphans_hvac: [30:111111, 32:333333, 32:555555, 32:666666]\n```",
+                    "known_list": "A mapping of known device IDs and optionally their device classes. For example:\n```\n18:123456: '{}' # Honeywell HGI80\n30:111111: '{class: FAN}' # Orcon HRU\n32:333333: '{class: REM, faked: true}' # an impersonatable remote\n32:555555: '{class: CO2, faked: true}' # a fully faked sensor\n32:666666: '{class: HUM}'\n```",
+                    "enforce_known_list": "Recommended once you have entered all your device IDs as the RAMSES II protocol does not include error correction and corrupt device IDs are common."
+                }
+            },
+            "advanced_features": {
+                "title": "Advanced features",
+                "data": {
+                    "send_packet": "Allow sending packets",
+                    "message_events": "Emit events for messages matching regular expression"
+                },
+                "data_description": {
+                    "message_events": "Enter a regular expression against which to match messages. For example `RP.* 10:` will emit events for all messages sent by an OTB."
+                }
+            },
+            "packet_log": {
+                "title": "Packet log",
+                "description": "Optional packet log to aid troubleshooting and help development.",
+                "data": {
+                    "file_name": "Packet log file name",
+                    "rotate_bytes": "Maximum size of each packet log",
+                    "rotate_backups": "Number of packet logs to keep"
+                },
+                "data_description": {
+                    "file_name": "Path to a packet log file. Retained backups will use this filename with a suffix."
+                }
+            }
+        }
+    },
+    "options": {
```

Comment:
Annoyingly there is no way to inherit the translations from the config section so this duplicates the keys for the shared dialogs.